packages/wonder-blocks-progress-spinner/components/circular-spinner.js

```diff
@@ -0,0 +1,114 @@
+// @flow
+import * as React from "react";
+import {StyleSheet} from "aphrodite";
+
+import {View, addStyle} from "wonder-blocks-core";
+import Color from "wonder-blocks-color";
+
+const heights = {
+    xsmall: 16,
+    small: 48,
+    default: 96,
```

Comment:
These will need to change once we get the 24x24 side which matches the default icon size.

Reply:
Instead of having a `default` size I think we should continue with the same `xsmall, small, medium, large, xlarge` that we've been use elsewhere.  We can use `defaultProps` to pick a "default".

Reply:
Maybe:
```
xsmall: 16,
small: 24,
medium: 48,
large: 96,
```

Reply:
To clarify: this will mean that we can no longer switch existing ProgressSpinners to use this new one as they currently correlate to the sizes that I used here. I can change it, but it will just make the rollout more challenging.